agency/README.md

```diff
@@ -222,6 +175,23 @@ List of valid vehicle events and the resulting vehicle status if the event is su
 | `maintenance_pick_up`  | Vehicle removed from street so it can be worked on                                                   | `available`, `unavailable`                         | `removed`           |                                                                         |
 | `deregister`           | A vehicle is deregistered                                                                            | `available`, `unavailable`, `removed`, `elsewhere` | `inactive`          | A vehicle is deactivated from the fleet and unavailable.                |
 
+## Telemetry Data
+
+A standard point of vehicle telemetry. References to latitude and longitude imply coordinates encoded in the [WGS 84 (EPSG:4326)](https://en.wikipedia.org/wiki/World_Geodetic_System) standard GPS projection expressed as [Decimal Degrees](https://en.wikipedia.org/wiki/Decimal_degrees).
+
+| Field          | Type           | Required/Optional     | Field Description                                            |
+| -------------- | -------------- | --------------------- | ------------------------------------------------------------ |
+| `vehicle_id`   | UUIDv4         | Required              | ID used in [Register](#vehicle-register)                     |
+| `timestamp`    | Unix Timestamp | Required              | Date/time that event occurred. Based on GPS clock            |
```

Comment:
if it's from the GPS shouldn't it be  `gps.timestamp` ?

Reply:
I believe the rationale is that we may at some point have non-gps telemetry, either from a vehicle, or from external sensors that aren't necessarily GPS-based.  Also it's nice to have `timestamp` at the top level of the object, as with events.